Exports issue, common js, path resolution

I am having trouble when using the command component in shadcn-svelte and believe it may be caused by cmdk-sv.  

`ReferenceError: exports is not defined
    at /node_modules/cmdk-sv/dist/index.js:25:23`

I've tried a number of fixes aimed at handling commonjs:
- vite config optimizeDeps.include for cmdk-sv
- vite-plugin-commonjs

When commonjs fixes are applied i run into issues with resolving files in the $lib path:

✘ [ERROR] Could not resolve "$lib/internal/command-score.js"
✘ [ERROR] Could not resolve "$lib/internal/index.js"


A recent change updated the package to sveltekit 2.0, I cannot determine if it is related to this. 


To replicate this issue:
- create sveltekit project with defaults
- shadcn-svelte with defaults
- npx to add command component
- use command component
- issue resolves when cmdk-sv is commented out.

--- a/.changeset/slimy-insects-worry.md
+++ b/.changeset/slimy-insects-worry.md
@@ -1,0 +1,5 @@
+---
+'cmdk-sv': patch
+---
+
+chore: Updated `svelte-package` to fix malformed package build

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
 		"@playwright/test": "^1.28.1",
 		"@sveltejs/adapter-vercel": "^4.0.0",
 		"@sveltejs/kit": "^2.0.0",
-		"@sveltejs/package": "^2.2.2",
+		"@sveltejs/package": "^2.2.7",
 		"@sveltejs/vite-plugin-svelte": "^3.0.0",
 		"@svitejs/changesets-changelog-github-compact": "^1.1.0",
 		"@types/prismjs": "^1.26.2",

